logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/LogStreamImpl.java

```diff
@@ -135,11 +136,25 @@ public void close() {
 
   @Override
   protected void handleFailure(final Throwable failure) {
```

Comment:
:wrench: Could we extract an internal method that we can then call from `openAppender`?  For me it's a bit confusing that we are calling an actor method manually, `LogStreamImpl` would be the only class where we do this.

Reply:
Done